test/e2e/network/netpol/kubemanager.go

```diff
@@ -66,7 +65,7 @@ func newKubeManager(framework *framework.Framework) *kubeManager {
 func (k *kubeManager) initializeCluster(model *Model) error {
 	var createdPods []*v1.Pod
 	for _, ns := range model.Namespaces {
-		_, err := k.createNamespace(ns.Spec())
+		_, err := k.framework.CreateNamespace(ns.Spec().Name, map[string]string{})
```

Comment:
This omits the label selector of `ns.Spec()` of the original NS creation, I wonder whether this was intended.

Also, a small nit: you can do just `ns.Name`.

All and all, I think the following might work a little better:
```golang
_, err := k.framework.CreateNamespace(ns.Name, ns.LabelSelector())
```

Reply:
Ack, Done!